python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -539,7 +539,11 @@ def makeGraph(self, pipeline, args):
             # make execution plan (a.k.a. DAG) for pipeline
             graphBuilder = GraphBuilder(registry,
                                         skipExisting=args.skip_existing)
-            qgraph = graphBuilder.makeGraph(pipeline, collections, run, args.data_query)
+            # accumulate metadata
+            metadata = {"input": args.input, "output": args.output, "butler_config": args.butler_config,
+                        "output_run": args.output_run, "extend_run": args.extend_run,
+                        "skip_existing": args.skip_existing}
```

Comment:
On JIRA @timj also asked about few other items: time of creation, user name, data query (should be in `args.data_query`). I am somewhat concerned that `butler_config` may point to some temporary location which could disappear, should we be storing registry database URL just in case?

Reply:
I think the butler URI for the repository itself is probably better than the `ButlerConfig` (or at least we should definitely include the butler URI regardless even if we think we also need the config itself).

Reply:
`butler_config` here is just a command line argument which is a path/URL, not the whole `ButlerConfig`.

Reply:
Okay. That's not a good name for it then.

Reply:
Indeed, historically it comes from `--butler-config` command line option which by default was translated to `butler_config` attribute name by `argparse` (which we don't use any more).

Reply:
I changed the key in the metadata to say "butler_argument" to better reflect that it was what was passed on the command line, and side step the name `butler_config`